doc/api/os.md

```diff
@@ -215,6 +215,15 @@ added: v2.3.0
 The `os.homedir()` method returns the home directory of the current user as a
 string.
 
+POSIX:
```

Comment:
```suggestion
**POSIX**:
```

Reply:
Thank you!, fixed on [a2f0be8](https://github.com/nodejs/node/pull/28401/commits/a2f0be843a5390b870960948d654fc8c75538a9a)